src/Error/ExceptionTrap.php

```diff
@@ -222,17 +221,7 @@ public function handleException(Throwable $exception): void
         }
         $request = Router::getRequest();
 
-        $shouldLog = true;
-        foreach ($this->_config['skipLog'] as $class) {
```

Comment:
Don't we still need this for global exception handling? Should we move this logic to `logException()` instead of removing it entirely?

Reply:
We can just deal with it being passed to the logger and exclude it there. I thought this would be a cleaner top-level filter, but seems not.